### ISSUE TITLE
[help] How to add new chickens with the chickens.json?

### ISSUE THREAD

**Issue by EnderiteFox** [CreeperHost/Chickens#40]
Hi,
So I've tried to add new chickens for the Mekanism ores, but it just doesn't work, can you tell me how I can do that?

Here's what I added to the chickens.json:
```
"chickens:osmium_chicken": {
    "name": "osmium_chicken",
    "is_enabled": true,
    "lay_coefficient": 1.0,
    "lay_item": {
      "itemID": "mekanism:ingot_osmium"
    },
    "drop_item": {
      "itemID": "mekanism:ingot_osmium"
    },
    "spawn_type": "NORMAL",
    "parent_1": "chickens:iron_chicken",
    "parent_2": "chickens:quartz_chicken"
  },
  "chickens:uranium_chicken": {
    "name": "uranium_chicken",
    "is_enabled": true,
    "lay_coefficient": 1.0,
    "lay_item": {
      "itemID": "mekanism:ingot_uranium"
    },
    "drop_item": {
      "itemID": "mekanism:ingot_uranium"
    },
    "spawn_type": "NORMAL",
    "parent_1": "chickens:osmium_chicken",
    "parent_2": "chickens:emerald_chicken"
  }
```

**Comment by InfinitysDaughter** [comment]
Have you made the resource pack to go with the new chickens?

**Comment by EnderiteFox** [comment]
> Have you made the resource pack to go with the new chickens?

No, so I need to make one?
I checked and the chickens aren't there, even in JEI

**Comment by InfinitysDaughter** [comment]
Ya. Its not stated in the text file but I saw that in an old closed thread about adding mod support.

But I think there is something keeping the code from registering cuse I've got the same problem and I do have a resource pack.

**Comment by JCAustin** [comment]
Curious to see if anyone has been successful in adding custom chickens in the latest release. I'm pretty sure I've got everything added, updated chickens.json and added the assets but nothing shows up in game and no errors.

**Comment by gigabit101** [comment]
What Minecraft versions?

**Comment by EnderiteFox** [comment]
1.19.2, Forge-43.2.0, Mod version 1.0.8

**Comment by gigabit101** [comment]
Fixed in 1.19.2-1.0.30
